kymatio/scattering2d/tests/test_tensorflow_scattering2d.py

```diff
@@ -0,0 +1,74 @@
+import os
+import io
+import tensorflow as tf
+from kymatio.scattering2d import Scattering2D
+import torch
+import numpy as np
+import pytest
+from collections import namedtuple
+
+class TestScattering2DTensorflow:
```

Comment:
Add more tests

Reply:
I agree - I'm not a good candidate to this task - I don't understand much of tf!

Reply:
Well, we should have at least the same types of tests that we do for torch. Putting code out that we don't test because we don't know how to test it sounds like a bad idea.

Reply:
Resolve?

Reply:
OK - so we need to add that in frontend

Reply:
I don't know which tests to add

Reply:
Wait, why is this class repeated?

Reply:
fixed